### ISSUE TITLE
Make `cell requested float voltage` configurable

### ISSUE THREAD

**Issue by syssi** [syssi/esphome-jk-bms#422]
1. Vol. Cell RFV   Float voltage
2. RCV time (h)   Absorption time
3. RFV time (h)   Float voltage timer

**Comment by syssi** [comment]
```
# RVC V
aa:55:90:eb: 09 :04: 7f:0d:00:00 :36:d5:24:4a:68:8e:ad:95:5f:23 # 3455 = 3.455V
aa:55:90:eb: 09 :04: 7e:0d:00:00 :3c:35:b5:c4:8c:dd:6c:11:32:14 # 3454 = 3.454V

# RFV V
aa:55:90:eb: 0a :04: 49:0d:00:00 :d6:bb:2e:15:63:0e:3c:dc:a4:df # 3401 * 0.001f = 3.401V
aa:55:90:eb: 0a :04: 48:0d:00:00 :3a:7a:06:20:a7:93:1b:34:dd:1d # 3400 * 0.001f = 3.400V

# RCV time
aa:55:90:eb: b3 :01: 14 :71:55:0b:ba:97:e6:68:67:fe:71:5b:50:33 # 20 * 0.1f = 2.0h
aa:55:90:eb: b3 :01: 0a :76:55:c2:22:63:4f:02:ce:a8:d8:a7:0a:9a # 10 * 0.1f = 1.0h

# RFV time
aa:55:90:eb: b4 :01: 5a :45:6a:43:1a:4d:85:5f:3e:db:6a:f8:73:b4 # 90 * 0.1f = 9.0h
aa:55:90:eb: b4 :01: 50 :d1:61:5a:d0:53:35:21:c9:8a:63:ec:6d:93 # 80 * 0.1f = 8.0h
```

**Comment by syssi** [comment]
Fixed by #485
